docs/src/main/sphinx/connector/mysql.md

```diff
@@ -276,6 +276,22 @@ this table:
 
 No other types are supported.
 
+### Timestamp type handling
+
+MySQL `TIMESTAMP` types with are mapped to a Trino `TIMESTAMP WITH TIME ZONE`.
+To preserve time instants, Trino's use of the MySQL JDBC driver sets the JVM
+time zone as the session zone and forces the server's time zone to match the
+JVM time zone. Because of this, users may experience an issue if the JVM time
```

Comment:
I'm not a native English speaker, but the order of this sentence makes it hard to understand. How about this?
```suggestion
To preserve time instants, Trino sets the session time zone
of the MySQL connection to match the JVM time zone.
Because of this, users may experience an issue if the JVM time
```

Reply:
This is much more clear, thank you.